lib/utils/fsUtils.js

```diff
@@ -454,13 +454,23 @@ function getTemplateId(type, envId, templateConfig) {
   switch (type) {
     case "firm":
       return templateConfig?.id?.[envId];
-      break;
     case "partner":
       return templateConfig?.partner_id?.[envId];
-      break;
     default:
       throw "Invalid template type: ${type}";
-      break;
+  }
+}
+
+function setTemplateId(type, envId, templateConfig, templateId) {
+  switch (type) {
+    case "firm":
+      templateConfig.id[envId] = templateId;
+      return templateConfig.id[envId];
+    case "partner":
+      templateConfig.partner_id[envId] = templateId;
+      return templateConfig.partner_id[envId];
+    default:
+      throw "Invalid template type: ${type}";
```

Comment:
Why `break` statements are return? I believe they should be there. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch